spring-kafka/src/main/java/org/springframework/kafka/support/serializer/ErrorHandlingDeserializer2.java

```diff
@@ -104,8 +104,8 @@ public void setFailedDeserializationFunction(BiFunction<byte[], Headers, T> fail
 		setFailedDeserializationFunction((failed) -> failedDeserializationFunction.apply(failed.getData(), failed.getHeaders()));
 	}
 	/**
-	 * Set in order to have a supplier for a T value when deserialization fails.
-	 * @param failedDeserializationFunction Function type param to he invoked.
+	 * Called this method passing a Function<FailedDeserializationInfo, T> in order to have a supplier for a T value when deserialization fails.
```

Comment:
Well, we prefer method Java in the imperative style: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

So, `Set` is OK, but not enough info what to set.